umbraco-cloud/getting-started/migrate-existing-site/README.md

```diff
@@ -196,15 +196,16 @@ In this next part, it is time to generate the so-called UDA-files for all your p
 
 For more details about UDA files, read the [UDA Files](../../set-up/power-tools/generating-uda-files.md#what-are-uda-files) article.
 
+Your local Cloud project will need [Umbraco Deploy](https://www.nuget.org/packages/Umbraco.Deploy.OnPrem) installed, you can check your .csproj file to see if this is already installed. If not installed, install this and check your site is working as expected.
+ 
 * Make sure the folder `/Umbraco/Deploy/Revision` on your Umbraco Cloud project is empty
   * If you have any files in the folder, you can safely remove those at this point
 * Start your local cloud project
 * Go to the settings section in your project
-* Select `Extract Schema To Data Files` Command from the **Deploy Operations** drop-down and click **Trigger Operation**.
+* Go to the Deploy tab
+* Under **Deploy Operations** > **Export Schema To Data Files** click **Export Schema**.
   * Generating the UDA files may take a while, depending on the extent of your project
-  * You will see a `Last deployment operation completed` status once the export is done
-  * Run `Schema Deployment From Data Files` to check that the UDA files have been generated correctly
```

Comment:
I think this should still be a part of the guide, the only thing is to rename it to: `Run `Update Umbraco Schema` to check that the UDA files have been generated correctly` as it is to just make sure that everything is generated correctly 😊

Reply:
Have made that change @jonat123